workflows/run_docker_server.py

```diff
@@ -111,6 +112,25 @@ def run_docker_server(model_spec, setup_config, json_fpath):
         "TT_MODEL_SPEC_JSON_PATH": docker_json_fpath,
     }
 
+    # Add environment variables for tt-media-server containers (audio, image, cnn models)
```

Comment:
```suggestion
    # TODO: Refactor this to set all of these variables inside the ModelSpec json
    # Add environment variables for tt-media-server containers (audio, image, cnn models)
```

We should add a note for our future selves that we need to abstract this into the ModelSpec json + make media server use the ModelSpec json